tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -201,6 +201,39 @@ def test_fn_to_docker_container(self, ondemand_aws_cluster):
     ####################################################################################################
     # Status tests
     ####################################################################################################
+
+    @pytest.mark.level("minimal")
+    @pytest.mark.clustertest
+    def test_status_scheduler_basic_flow(self, cluster):
+
+        cluster.save()
+
+        cluster_logs = cluster.run([f"cat {SERVER_LOGFILE_PATH}"], stream_logs=False)[
```

Comment:
just noting we might be able to replace this with a call to the new /logs endpoint, which might be ready soonish

Reply:
added a TODO comment as a reminder for that.